sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2252,6 +2252,8 @@ pi_result cuda_piQueueFinish(pi_queue command_queue) {
   return result;
 }
 
+pi_result cuda_piQueueFlush(pi_queue command_queue) { return PI_SUCCESS; }
```

Comment:
Is there no CUDA equivalent that forces the flush? At least, add a comment explaining the implementation